pyazul/index.py

```diff
@@ -99,8 +99,7 @@ def post_sale_transaction(self, data):
         return self.azul_request(data, operation='ProcessPost')
 
     def verify_transaction(self, data):
-        data.update(validate.verify_transaction(
-            data, operation='VerifyPayment'))
+        data.update(validate.verify_transaction(data))
         return self.azul_request(data)
```

Comment:
'operation' parameter was in the wrong place, this is what it should look like.
`
return self.azul_request(data, operation='VerifyPayment')
`